corehq/apps/users/dbaccessors/all_commcare_users.py

```diff
@@ -43,6 +45,8 @@ def get_commcare_users_by_filters(domain, user_filters, count_only=False):
         query = query.role_id(role_id)
     if search_string:
         query = query.search_string_query(search_string, default_fields=['first_name', 'last_name', 'username'])
+    if location_id:
+        query = query.location(location_id)
```

Comment:
Actually this might not work quite as expected - it filters by the location selected, but I'd guess the desired behavior is to include users at all child locations as well.  That is, if you select "Massachusetts", you should also see users assigned to Boston.  To do that, it'll need to be something like:
```python
location_ids = SQLLocation.objects.get_locations_and_children_ids([location_id])
query = query.location(location_ids)
```
or
```python
location = SQLLocation.objects.get(location_id=location_id)
query = query.location(location.get_descendants(include_self=True).location_ids())
```

Reply:
Great point @esoergel. I totally missed that.
Would let product or jenny chime in, but i think this probably is not needed here even though that resonates with how we filter by locations at other places on HQ.

Reply:
Pinged Derek on the ticket.

Reply:
updated to include users assigned to children of the selected location: d2af9b83d35